src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -224,6 +208,10 @@ private boolean isSameOrBefore(Loan loan, DateTime proposedDueDate) {
       || proposedDueDate.isBefore(loan.getDueDate());
   }
 
+  private boolean numberOfRenewalsMatchLimit(Loan loan) {
```

Comment:
```suggestion
  private boolean hasReachedRenewalLimit(Loan loan) {
```

Reply:
Fix

Reply:
It seems to me that the expression `that limit has been reached` is more understandable. And this is not so misleading as the expressions: `has renewals remaining` and `has no renewals remaining`

Reply:
How does `the limit has been reached` fit together with `there is no limit`?

I'm comfortable with that terminology if we can find a way to fit those together.

Reply:
I'm comfortable with the naming changes you have made